tests/wpunit/includes/Standard_Source_Test.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace ISC\Tests\WPUnit;
+
+use ISC\Includes\Standard_Source;
+use phpDocumentor\Reflection\Types\Object_;
+
+/**
+ * Testing the Standard_Source class
+ */
+class Standard_Source_Test extends \Codeception\TestCase\WPTestCase {
+
+	/**
+	 * User ID
+	 *
+	 * @var int
+	 */
+	protected $user_id;
+
+	/**
+	 * Cleanup
+	 */
+	protected function tearDown(): void {
+		// Delete the user created in test_get_standard_source_text_for_attachment_post_author
+		if ( isset( $this->user_id ) ) {
+			wp_delete_user( $this->user_id );
+		}
+
+		// Reset modified options
+		delete_option( 'isc_options' );
+
+		// Call parent tearDown
+		parent::tearDown();
+	}
+
+	/**
+	 * Test the get_standard_source_text() method
+	 *
+	 * @return void
+	 */
+	public function test_get_standard_source_text() {
+		$this->assertEquals( '© http://isc.local', Standard_Source::get_standard_source_text() );
+	}
+
+	/**
+	 * Test the get_standard_source() method to see which option is selected by default (should be Custom Text)
+	 *
+	 * @return void
+	 */
+	public function test_get_standard_source() {
+		$this->assertEquals( 'custom_text', Standard_Source::get_standard_source() );
+	}
+
+	/**
+	 * Test the standard_source_is() method to see which option is selected by default (should be Custom Text)
+	 *
+	 * @return void
+	 */
+	public function test_standard_source_is() {
+		$this->assertTrue( Standard_Source::standard_source_is( 'custom_text' ) );
+	}
+
+	/**
+	 * Test the use_standard_source() method
+	 * This should return false, because the standard source is not used for the given attachment
+	 *
+	 * @return void
+	 */
+	public function test_use_standard_source() {
+		$this->assertEmpty( Standard_Source::use_standard_source( 123 ) );
+	}
```

Comment:
**Suggestion:** Consider using `assertNull` instead of `assertEmpty` for `test_use_standard_source` method. This will make your test more precise as it will fail if the method under test returns anything other than `null`. [best practice]
```suggestion
 public function test_use_standard_source() {
     $this->assertNull( Standard_Source::use_standard_source( 123 ) );
 }
```